src/main/java/com/kryptnostic/conductor/rpc/Lambdas.java

```diff
@@ -33,4 +41,28 @@ public static Runnable foo() {
         return (Function<ConductorSparkApi, QueryResult> & Serializable) ( api ) -> api
                 .getAllEntitiesOfEntitySet( entityFqn, entitySetName, authorizedProperties );
     }
+    
+    public static Function<ConductorSparkApi, Boolean> submitEntitySetToElasticsearch( EntitySet entitySet, List<PropertyType> propertyTypes, Principal principal ) {
```

Comment:
This is getting pretty ugly (due to past choices) we may have to modify how  we are grouping these together separate from this PR.